Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM  adoptopenjdk/openjdk11:alpine AS build
+WORKDIR /app
+COPY . ./
+RUN ./gradlew clean build
+
+FROM adoptopenjdk/openjdk11:alpine AS unit-test
+WORKDIR /app
+COPY --from=build /app/src .
+RUN ./gradlew clean check
+
+FROM adoptopenjdk/openjdk11:alpine AS beacons-service
+COPY --from=build /app/build/libs/*.jar app.jar
```

Comment:
I would set a `WORKDIR` like you have for the other stages here so that we aren't placing/executing the JAR from the root of the image. Probably `WORKDIR /opt/app` would make most sense, but `WORKDIR /usr/local/app` would work as well.  [See here](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard).  This will also mean that in the `ENTRYPOINT` command you shouldn't have to do `/app.jar` but rather `app.jar`